lib/collections/addon/index/template.hbs

```diff
@@ -62,13 +62,15 @@
                         </a>
                         <div class='m-t-md'>
                             {{!-- (route-prefix 'content') --}}
-                            {{#link-to
-                                'discover'
-                                (if this.theme.provider.example this.theme.provider.example 'khbvy')
-                                invokeAction=(action 'click' 'link' 'Index - See example')
-                            }}
+                            <LinkTo
+                                @route='discover'
+                                @models={{array
+                                    (if this.theme.provider.example this.theme.provider.example 'khbvy')
+                                }}
+                                invokeAction={{action 'click' 'link' 'Index - See example'}}
```

Comment:
Was curious what `invokeAction` is about, and it seems like we don't even have [ember-invoke-action](https://emberobserver.com/addons/ember-invoke-action) as a dependency or in our yarn.lock file. Seems like this was [ripped right from ember-osf-preprints](https://github.com/CenterForOpenScience/ember-osf-preprints/blob/develop/app/templates/index.hbs#L48) and there's not a collections index page. I think it's safe to remove this line